scapy/contrib/cansocket_native.py

```diff
@@ -28,6 +28,7 @@
 
 class CANSocket(SuperSocket):
     desc = "read/write packets at a given CAN interface using PF_CAN sockets"
+    async_select_unrequired = True
```

Comment:
That's a strange name =)

Reply:
Renamed `nonblocking_socket`, should be clearer :smile: